docs/diagrams/AddAttendanceSequenceDiagram.puml

```diff
@@ -14,10 +14,10 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("add-attendance c\1 l\1 s\1")
+[-> LogicManager : execute()
```

Comment:
I think for sequence diagrams it might be better to leave the arguments in since the original example from AB-3 had them.